manifest.yaml

```diff
@@ -259,6 +259,32 @@ packages:
  # https://github.com/openshift/machine-config-operator/pull/2421
  - conntrack-tools
 
+repo-packages:
+  # we always want the kernel from BaseOS
+  - repo: rhel-8-baseos
+    packages:
+      - kernel
+  # we want the one shipping in RHEL, not the equivalently versioned one in RHAOS
+  - repo: rhel-8-appstream
+    packages:
+      - nss-altfiles
+  - repo: rhel-8-server-ose
+    packages:
+      # eventually, we want the one from the container-tools module, but we're
+      # not there yet
+      - toolbox
+      # we want these from RHAOS to make sure they're in sync
+      # https://github.com/openshift/installer/pull/3062#issuecomment-582614009
+      - cri-o
+      - conmon
+
+modules:
+  enable:
+    # podman stack
+    - container-tools:rhel8
```

Comment:
While I'm glad we are able to select the module we use in the manifest, we should consider the support stance of each stream (i.e. 2.0/3.0/rhel8).  I don't want us to be in a position where we have selected a stream that is not going to be supported for the entirety of the 4.9 lifecycle.

Reply:
Hmm, my understanding is that `rhel8` is the moving target which should be valid for all of el8. It's essentially what we've implicitly been following since its packages are usually the latest NEVRA, which is what rpm-ostree has been picking so far being modularity-unaware. (In fact, [in a few days](https://errata.devel.redhat.com/advisory/79166) we'll be bumping to podman 3.2.3 once the module is updated.) So this is just matching current reality.

But yeah, we should probably get in touch with the containers team on this to clarify things.

Reply:
>  So this is just matching current reality.

That's enough to satisfy my concerns.  Thanks, Jonathan!